backend/internal/supabase/migrations/20240917002108_base_schema.sql

```diff
@@ -60,4 +60,38 @@ create table if not exists review (
     comment text,
     created_at timestamp with time zone default now(),
     updated_at timestamp with time zone default now()
+);
+
+create table if not exists follower (
+    follower_id uuid not null references "user"(id),
+    followee_id uuid not null references "user"(id),
+    primary key (follower_id, followee_id)
+);
+
+create table if not exists playlist (
+    playlist_id serial primary key,
+    title text not null,
+    user_id uuid not null references "user"(id),
+    bio text,
+    cover_photo text
+);
+
+create table if not exists playlist_track (
+    playlist_id bigint not null references playlist(id),
+    track_id bigint not null references track(id),
+    primary key (playlist_id, track_id)
+);
+
+create table if not exists user_auth (
+    id serial primary key,
+    user_id uuid not null references "user"(id),
+    refresh_token text not null,
+    access_token text not null,
+)
```

Comment:
this is dumb but i think if you don't include a semicolon after the paren here it'll freak out